README.md

```diff
@@ -166,6 +166,7 @@ Emogrifier currently support the following
  * attribute only
  * first-child
  * last-child
+ * not
```

Comment:
In the latest README, I've linked the corresponding MDN entry and also sorted the supported selectors the same way they are sorted in MDN. When you rebase, could you also do this for the `not` selector (which is a pseudo-class)? Thanks!